server/src/main/java/org/apache/iotdb/db/engine/storagegroup/TsFileProcessor.java

```diff
@@ -647,6 +650,7 @@ public void flushOneMemTable() {
 
   private void endFile() throws IOException, TsFileProcessorException {
     long closeStartTime = System.currentTimeMillis();
+//    updateDeviceChunkPointSizeCache();
```

Comment:
if the comment is used, it shoule be deleted then

Reply:
okay